format/spec.md

```diff
@@ -128,13 +128,13 @@ Tables do not require rename, except for tables that use atomic rename to implem
 
 #### Writer requirements
 
-Some tables in this spec have columns that specify requirements for v1 and v2 tables. These requirements are intended for writers when adding metadata files to a table with the given version.
+Some tables in this spec have columns that specify requirements for v1 and v2 tables. These requirements are intended for writers when adding metadata/manifest files to a table with the given version.
```

Comment:
manifest list file also need to be mentioned? 

I thought previously `metadata` word is used as generic word for all Iceberg metadata files not table metadata here.

Reply:
> I thought previously metadata word is used as a generic word for all Iceberg metadata files not table metadata here.

I agree with you here, I tried to be more explicit to avoid confusion between the metadata as a concept and the metadata file. Adding manifest-list is a good idea 👍 Let's take the suggestion (https://github.com/apache/iceberg/pull/8672#discussion_r1339897353) below.